build.cake

```diff
@@ -354,12 +371,43 @@ private bool HasAuthenticodeSignature(FilePath filePath)
     }
 }
 
-void SignFiles(IEnumerable<FilePath> files, FilePath certificatePath, string certificatePassword, string display = "", string displayUrl = "")
+void SignFilesWithAzureSignTool(IEnumerable<FilePath> files, string vaultUrl, string vaultAppId, string vaultAppSecret, string vaultCertificateName, string display = "", string displayUrl = "")
+{
+  var signArguments = new ProcessArgumentBuilder()
+    .Append("sign")
+    .Append("--azure-key-vault-url").AppendQuoted(vaultUrl)
+    .Append("--azure-key-vault-client-id").AppendQuoted(vaultAppId)
+    .Append("--azure-key-vault-client-secret").AppendQuotedSecret(vaultAppSecret)
+    .Append("--azure-key-vault-certificate").AppendQuoted(vaultCertificateName)
+    .Append("--file-digest sha256");
+
+  if (!string.IsNullOrWhiteSpace(display))
+  {
+    signArguments
+      .Append("--description").AppendQuoted(display)
+      .Append("--description-url").AppendQuoted(displayUrl);
+  }
+
+  foreach (var file in files)
+    signArguments.AppendQuoted(file.FullPath);
+
+    var azureSignToolPath = MakeAbsolute(File("./tools/Cake/azuresigntool.exe"));
```

Comment:
Interestingly the path is `./tools/Cake/azuresigntool.exe` in octopus server but `./tools/azuresigntool.exe` in tentacle?

Reply:
Good catch... Eagle-Eye-Corey.
Looking locally... it should be `tools/azuresigntool.exe`
```suggestion
    var azureSignToolPath = MakeAbsolute(File("./tools/azuresigntool.exe"));
```